### ISSUE TITLE
[aws-lambda] Add "Topics" property to EventSourceMappingOptions

### ISSUE THREAD

**Issue by dscpinheiro** [aws/aws-cdk#10138]
Lambda recently added support for MSK as an event source (https://aws.amazon.com/about-aws/whats-new/2020/08/aws-lambda-now-supports-amazon-managed-streaming-for-apache-kafka-as-an-event-source/), and there's now a "Topics" property on the CloudFormation resource definition (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-eventsourcemapping.html#cfn-lambda-eventsourcemapping-topics).

### Use Case
Using Lambda simplifies reading data from Apache Kafka, as other options usually require customers to worry about infrastructure management.

### Proposed Solution
Include `topics` as an optional parameter of `EventSourceMappingOptions` interface:

```typescript
export interface EventSourceMappingOptions {
    /**
     * existing properties
     */

    /**
     * The name of the Kafka topic.
     * 
     * Valid range:
     * * Maximum value of 1 
     */
    readonly topics?: string[];
}
```

It'd then be used as:
```typescript
lambdaFn.addEventSourceMapping('MSKTrigger', {
    eventSourceArn: 'arn:aws:kafka:us-east-1:123456789012:cluster/kafka-cluster/random-id',
    startingPosition: lambda.StartingPosition.LATEST,
    topics: ['my-topic']
});

```

### Other
In the future, there could even be a `MSKEventSource` in the `aws-lambda-event-sources` package, but just adding the `topics` property should allow customers to get started.

* [x] :wave: I may be able to implement this feature request
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

**Comment by dscpinheiro** [comment]
So it looks like `CfnEventSourceMapping` does not contain `topics` yet, but it's included in this PR (https://github.com/aws/aws-cdk/pull/9979).

@nija-at Is there an estimate when that PR will be merged?

**Comment by nija-at** [comment]
There is currently no higher level construct support for MSK in the CDK - https://github.com/aws/aws-cdk/tree/master/packages/%40aws-cdk/aws-msk

It will make sense for this to be added once we have some basic L2 support.

**Comment by dscpinheiro** [comment]
@nija-at This might be a dumb question, but wouldn't it still make sense to still add the `topics` property to the interface? The `EventSourceMappingOptions` interface would then match completely with the `AWS::Lambda::EventSourceMapping` resource type.

**Comment by nija-at** [comment]
Yes, you are right. This can absolutely be done.

Usually, this is accompanied by an integration class in [`aws-lambda-event-sources` package](https://docs.aws.amazon.com/cdk/api/latest/docs/aws-lambda-event-sources-readme.html). This will have to wait until the higher level construct for MSK is in place.

**Comment by dscpinheiro** [comment]
If I created the PR just adding the `topics` property, would that be rejected? My concern is that if a customer is already using the CDK to manage their MSK clusters, and now wants to add an integration with Lambda, that's just not possible (and waiting for a higher level construct might make them take a different approach).

I did find a PR for a L2 construct for MSK, but it hasn't been reviewed yet (https://github.com/aws/aws-cdk/pull/9908).
